packages/perseus/src/widgets/transformer.tsx

```diff
@@ -2521,6 +2521,7 @@ class Transformer extends React.Component<Props> {
                     // NOTE(kevinb): It's hard to convince that this.dilationCircle
                     // will still be defined here.                    center: self.dilationCircle.centerPoint.coord,
```

Comment:
the code you need is actually here in this comment. the newline got accidentally deleted